### PR TITLE
Upgrade dependencies to match quality checklist

### LINKED ISSUE #80: Code Exchange quality checklist
The purpose of this checklist is to assess the state of this repository, and bring it to an acceptable level for long-term support by completing the tasks listed below.

**This checklist is used internally but can be used as a guidance for others**

> Please check every part of this checklist for us to process your request. You can do this by marking them with [x] instead of [ ].

### Documentation

- [x] `LICENSE` file exists in repository

- [x] [Code of Conduct](https://github.com/twilio-labs/.github/blob/master/CODE_OF_CONDUCT.md) exists in repository

- [x] [CONTRIBUTING](https://github.com/twilio-labs/.github/blob/master/CONTRIBUTING.md) exists in repository 

- [x] No broken links in README.md

- [x] Build status exists at top of README.md

- [x] Set up steps are complete and accurate

- [x] CodeExchange repository is linked in README.md

### GitHub

- [x] Dependabot set up

- [x] Dependabot automerge set up

- [x] Uses GitHub Actions (required for Twilio sample) or other CI

### Code

- [x] Packages/dependencies are up to date

- [x] Code compiles against latest packages/dependencies

- [x] Matches [CodeExchange Guidelines](https://github.com/twilio-labs/code-exchange/tree/master/guidelines) to the best of ability

### Tests

- [x] Functionality tests have been created

- [x] All tests pass

### App

- [x] Confirmed sample behaves as expected (after dependencies have been updated)

- [x] This file has been copied into a wiki entry for the respective repository or added as a GitHub issue

### General

- [x] App is released under a permissive license like MIT or Apache-2

- [x] I've read and followed the contribution guidelines

- [x] I've read and implemented my template according to the respective coding guidelines for my project

- [x] I agree and adhere to the Code of Conduct

- [x] The template has been authored by me / or my organization and will be kept up-to-date. If the template will not longer be maintained, I'll notify this project.

### PR DESCRIPTION
**Pull request by rojasTob**
Resolves #80

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,11 +56,13 @@ To run the app locally, first clone this repository and `cd` into its directory.
     npm install
     ```
 
-1. Copy the `.env_example` file to `.env`, and edit it to include your [Twilio API credentials](https://www.twilio.com/console) and the phone number and TwimL App Sid you made above. We use [python-dotenv](https://github.com/theskumar/python-dotenv) to load those variables automatically.
+1. Copy the `.env.example` file to `.env`, and edit it to include your [Twilio API credentials](https://www.twilio.com/console) and the phone number and TwimL App Sid you made above. We use [python-dotenv](https://github.com/theskumar/python-dotenv) to load those variables automatically.
```
Comment (bharatmurali) [inline README.md:59]: Fix numbers please :)

File: README.md [hunk 2]
```diff
@@ -56,11 +56,13 @@ To run the app locally, first clone this repository and `cd` into its directory.
     npm install
     ```
 
-1. Copy the `.env_example` file to `.env`, and edit it to include your [Twilio API credentials](https://www.twilio.com/console) and the phone number and TwimL App Sid you made above. We use [python-dotenv](https://github.com/theskumar/python-dotenv) to load those variables automatically.
```
Comment (bharatmurali) [inline README.md:59]: neither of the virtualenv steps above work for me

File: README.md [hunk 3]
```diff
@@ -79,13 +81,20 @@ To run the app locally, first clone this repository and `cd` into its directory.
     python manage.py runserver
```
Comment (bharatmurali) [inline README.md:81]: we should consider moving this to python3

File: README.md (hunk 2 above)
Comment (rojasTob) [reply README.md:59]: Hey Bharat. Thanks for pointing this out. I'm going to update the Readme file with the template and add commands from the Makefile.

**Comment by rojasTob** [comment]
Hi @bharatmurali, changes are done. Thanks!

File: README.md (hunk 3 above)
Comment (bharatmurali) [reply README.md]: @rojasTob I'm now getting `ModuleNotFoundError: No module named 'bootstrap4'`. Is this working for you?

Comment (rojasTob) [reply README.md]: Hey @bharatmurali. Yes, this is working in my local. However, I've modified the step `6.` in the Readme file, it seems like the virtual env was not activated.

**Pull request merged**